### PR TITLE
Bug 1309569 - don't show job end date if it never ends

### PR DESCRIPTION
**Pull request by jezdez**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-analysis-service/44)

### CODE REVIEW

**Comment by jezdez** [comment]
@maurodoglio r?

File: atmo/templates/atmo/spark-job-detail.html [hunk 1]
```diff
@@ -56,12 +56,14 @@
...
       <dt>End date</dt>
-      <dd>{{ job.end_date|default:"n/a" }}</dd>
-      <dt>Is enabled</dt>
-      <dd><span class="glyphicon glyphicon-{% if job.is_enabled %}ok text-success{% else %}remove text-danger{% endif %}" aria-hidden="true"></span></dd>
+      <dd>{{ job.end_date }}</dd>
+      {% endif %}
       <dt>Last run date</dt>
       <dd>{{ job.last_run_date|timesince|default:"n/a" }}</dd>
+      <dt>Is enabled</dt>
+      <dd><span class="glyphicon glyphicon-{% if job.is_enabled %}ok text-success{% else %}remove text-danger{% endif %}" aria-hidden="true"></span></dd>
```
Comment (maurodoglio) [inline atmo/templates/atmo/spark-job-detail.html]: I guess this change sneaked into this patch by mistake?

Comment (jezdez) [inline atmo/templates/atmo/spark-job-detail.html]: Nah, just moved it down from line 60

Comment (maurodoglio) [inline atmo/templates/atmo/spark-job-detail.html]: okey dokey

**Pull request merged**